### PR TITLE
Fixed CA1062 and CA1812

### PR DESCRIPTION
**Pull request by mertyeter**
Fixed **CA1062: Validate arguments of public methods** and **CA1812: Avoid uninstantiated internal classes**

### CODE REVIEW

File: src/SQLQueryStress/LoadEngine.cs [hunk 1]
```diff
@@ -558,46 +558,5 @@ public class QueryOutput
             }
              */
         }
-
```
Comment (ErikEJ) [inline src/SQLQueryStress/LoadEngine.cs:561]: Why was this removed?

Comment (ErikEJ) [reply src/SQLQueryStress/LoadEngine.cs:561]: And looks like very old code..

Comment (mertyeter) [reply src/SQLQueryStress/LoadEngine.cs:561]: It looks like it's not used by any of other classes, removed for "Avoid uninstantiated internal classes"

**Comment by ErikEJ** [comment]
Autumn cleaning!
